src/controllers/mintController.test.ts

```
@@ -0,0 +1,77 @@
+      res,
+      next,
+    );
+
+    const err = (next as jest.Mock).mock.calls[0][0] as AppError;
+    expect(err).toBeInstanceOf(AppError);
+    expect(err.statusCode).toBe(401);
+    expect(err.message).toBe("User context required for deposit");
+  });
+
+  it("rejects /mint/deposit when wallet_address does not match the user's stored wallet", async () => {
+    mockedUserFindUnique.mockResolvedValue({ stellarAddress: "GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF" });
```

ESLint

[error] Replace `·stellarAddress:·"GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF"` with `⏎······stellarAddress:⏎········"GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF",⏎···`

(prettier/prettier)